dracut/52fdo/manufacturing-client.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Manufacturing client DIUN
+DefaultDependencies=false
+
+After=coreos-installer.service
+Before=coreos-installer-reboot.service
+ConditionPathExists=/etc/manufacturing-client-config
+Requires=dev-disk-by\x2dlabel-boot.device
+
+OnFailure=emergency.target
+OnFailureJobMode=replace-irreversibly
+
+[Service]
+Type=oneshot
+EnvironmentFile=/etc/manufacturing-client-config
```

Comment:
Should we add `LOG_LEVEL=info` here as well?
Does systemd just append that to the contents it reads from the EnvironmentFile?